cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```diff
@@ -222,8 +222,35 @@ parameters:
 - name: CS_SERVICE_PRINCIPAL_CREDS_BASE64
   description: Base64 encoded service principal credentials for CS. This is temporary for P1
   required: true
+- name: KEYVAULT_NAME
```

Comment:
@bennerv will there be a case where these types of credentials/certificates are moved to their own key vault? 

I'd like to make the name of this key vault specific. For example we also have the arm helper identity credentials. Are these going to be stored in the same key vault? cc @miguelsorianod 

Maybe something like `AZURE_FIRST_PARTY_APPLICATION_CREDENTIALS_KEYVAULT_NAME` or `AZURE_IDENTITY_CREDENTIALS_KEYVAULT_NAME`?

Reply:
Would we like to create different key vaults to store different secrets? 
If so I'm ok with changing it, but this PR stores it in the private service-kv. Thus, I will propose (at this moment) to change it to `SERVICE_KEYVAULT_NAME`. What do you think?

Reply:
I think we should make the name meaningful to CS as the current kv name may change. It may also not have the same name for other environments. I think `SERVICE_KEYVAULT_NAME` is a bit ambiguous though, what does `SERVICE` mean? Is it related to cluster service, the service cluster, or something else? 🤔 

As for multiple kv vs one kv, I think @bennerv or @janboll is best to answer that 🙏

Reply:
Make sense, I will wait for the answer of multiple kv vs one, so we can select one of your suggestions. As one describes a general kv than the other.

Reply:
service comes from the name of the key vault used to support the service running the resource provider.  There will be other secrets in the service key vault which contain TLS certificates, MDSD certificates (potentially) etc.  

It might make sense to align on naming across the components so we're all talking about the same thing, but it's up to CS to determine what they want to do with the name.

Reply:
Then I'll opt for AZURE_IDENTITY_CREDENTIALS_KEYVAULT_NAME as it seems it will potentially contain other certs.